### PR TITLE
Tabs: Updated to DS6.v2.01	

### LINKED ISSUE #501: Tabs: update to DS6.v2.01
This new revision appears to add a solid grey line across the bottom of the component, and a new size:

screen shot 2019-01-08 at 11 22 09 am

screen shot 2019-01-08 at 11 21 54 am

### PR DESCRIPTION
**Pull request by ianmcburnie**
# Closes #501 

* Added large modifier (changes font size of tabs from 14px to 16px)
* Blue border thickness changed from 2px to 3px (ds6)
* Thin grey bar added, which sits in a layer underneath tabs.(ds6)
* Removed margin between tabs (ds6)

I had to juggle a few properties out of common, base styles into DS4 or DS6 specific files.

screen shot 2019-03-04 at 1 51 08 pm

### CODE REVIEW

File: src/less/tab/base/tab.less [hunk 1]
```diff
@@ -7,11 +7,17 @@ div.tabs__items[role="tablist"],
 ul.fake-tabs__items {
     display: flex;
     flex-direction: row;
+    font-size: 14px;
```
Comment (pcanella) [inline src/less/tab/base/tab.less]: Not sure, but are there LESS vars that can be used here or not?

File: src/less/tab/ds6/tab.less [hunk 2]
```diff
@@ -10,14 +10,30 @@
     border-bottom: 1px solid @ds6-color-white; // prevent margin-collapse on next element
 }
 
+// move the thin grey bar up by 1 pixel
+.tabs__content,
+.fake-tabs__content {
+    border-top: 1px solid #c7c7c7;
```
Comment (seangates) [inline src/less/tab/ds6/tab.less]: This grey is part of that [updated set we learned about this morning](https://github.com/eBay/skin/issues/580), and we do not yet have a variable for it. 😞

File: src/less/tab/ds6/tab.less [hunk 3]
```diff
@@ -10,14 +10,30 @@
...
+div.tabs__items[role="tablist"],
+ul.fake-tabs__items {
+    position: relative;
+    z-index: 1;
+}
+
 // outer
 div.tabs__item[role="tab"],
 li.fake-tabs__item {
+    border-bottom: 3px solid transparent;
```
Comment (seangates) [inline src/less/tab/ds6/tab.less]: In the fake-tabs case, why is the border on the `<li>` and not on the anchor? It makes for a weird focus ring which doesn't match the other tabs.

**Regular tab**

image

**Fake tab**

image

Is that just to simplify the CSS selectors? Or another reason I'm not understanding?

File: src/less/tab/base/tab.less (hunk 1 above)
Comment (ianmcburnie) [reply src/less/tab/base/tab.less]: Hard to spot in the diff, but:

```
// font-size should not scale with REM
```

File: src/less/tab/ds6/tab.less (hunk 3 above)
Comment (ianmcburnie) [reply src/less/tab/ds6/tab.less]: Hmm, not sure. I believe it was already like that and I just moved it out of base to ds6.

Comment (ianmcburnie) [reply src/less/tab/ds6/tab.less]: Changed it to anchor and it breaks the position of the gray bar in Safari.

Before change:

screen shot 2019-03-05 at 9 12 56 am


After change:

screen shot 2019-03-05 at 9 12 40 am

I'll maybe just leave it as it was for now and I can open a separate ticket to go on the backlog.

File: src/less/tab/base/tab.less (hunk 1 above)
Comment (pcanella) [reply src/less/tab/base/tab.less]: oh git it 👍

File: src/less/tab/ds6/tab.less (hunk 3 above)
Comment (seangates) [reply src/less/tab/ds6/tab.less]: Yeah, let's create a task. I had played with it and we will probably need some margin changes, etc., to accomplish it.

File: src/less/tab/ds6/tab.less (hunk 2 above)
Comment (ianmcburnie) [reply src/less/tab/ds6/tab.less]: Want me to take care of that task then? I thought you were doing it.

Comment (ianmcburnie) [reply src/less/tab/ds6/tab.less]: Oh, scrap that. We can't because it's a breaking change.

**Comment by ianmcburnie** [comment]
Issue for focus indicator: https://github.com/eBay/skin/issues/583

**Pull request merged**